charts/opentelemetry-kube-stack/examples/prometheus-otel/rendered/exporters/kube-scheduler/service.yaml

```diff
@@ -7,7 +7,8 @@ metadata:
   labels:
     app: opentelemetry-kube-stack-kube-scheduler
     jobLabel: kube-scheduler
-    helm.sh/chart: opentelemetry-kube-stack-0.3.3
+    
+    helm.sh/chart: opentelemetry-kube-stack-0.3.4
```

Comment:
why is this adding an extra space here?

Reply:
thanks @jaronoff97 for the review, this is fixed now.